packages/web/modals/wallet-select.tsx

```diff
@@ -28,6 +28,7 @@ import { Icon } from "~/components/assets";
 import { Button } from "~/components/buttons";
 import IconButton from "~/components/buttons/icon-button";
 import ClientOnly from "~/components/client-only";
+import { IntroducingOneClick } from "~/components/one-click-trading/introducing-one-click";
```

Comment:
Perhaps we can dynamically import this to keep the large SVG out of the main bundle